### ISSUE TITLE
The visual updates ruined this theme for me

### ISSUE THREAD

**Issue by VideoGirlAi** [darklinkpower/Harmony#37]
### Pre-Request Checklist

- [X] I have checked Playnite's settings (e.g., in Settings -> Appearance) and confirmed that this change cannot be made there.
- [X] I have checked that this feature is not already available using the [ThemeModifier plugin](https://playnite.link/addons.html#playnite-thememodifier-plugin).

### Feature Description

Harmony is my favorite Playnite theme, and I recently decided to update it from version 2.57 to 2.64, crossing my fingers that everything remained relatively the same in regards to the UI; but then, before even clicking "Update add-ons", I was disappointed when reading the changelog only to find the detail of "rounded icon corners" under version 2.63. Still, I was hoping I simply misunderstood what that meant.

Well, I did not misunderstand, and it turns out that that wasn't the only unfortunate change to Harmony; no, in fact, the UI underwent some rather drastic changes.

Thankfully, I had a backup on hand, and so I reverted back to version 2.57 (although it appears the UI changes weren't made until 2.60?), but I would love to be able to keep the theme updated with any improvements, so long as the UI that I grew comfortable with remains relatively the same.

And so, I come here to air my grievances with Harmony's updates and hope you can make accommodations, at least through the ThemeModifier plugin, or through providing a Legacy Layout version of Harmony (more on this Legacy idea at the end).

Without further ado:

1. I have my "Sidebar" on top (a setting found at "View > Sidebar > Top"). Now, in the updated Harmony, this centers the "Sidebar" buttons, much to my dismay. I feel like this is probably to keep the theme "modern" and in-line with Windows 11. However, I'm on Windows 10 still for many reasons, and I will admit, the centered taskbar does not make me want to update to Windows 11, to say the least. When I am finally forced to move to Windows 11 in Oct 2025, the first thing I will do is de-center the taskbar if it is within my power to do so. And so, it'd be great for the Sidebar buttons in Playnite to be de-centered, too, when in "Top" or "Bottom" mode.

2. The "Sidebar" and "Top panel" (hereby referred to as "Top Bar") are black now for some reason, clashing with my custom color scheme. You may think that black goes with everything, but it looked so much nicer, cleaner, and more uniform to have the Top Bar be the same colors as the rest of the theme, instead of a jarring black color.

3. The Top Bar being black also changed the color of all its non-selected buttons to gray; now, my custom Filter Preset buttons are the wrong color and I would really rather not have to edit them to be gray--a more boring color that I don't like.

4. The Top Bar is taller now. I don't see the reasoning for this; it was just fine before. It has approximately 24 pixels (this is a rough estimate) of unnecessary spacing at the bottom that it didn't have before. It looks off, and it takes up space that it doesn't need to take up.

5. Game icons are smaller in Details view (the actual details area, not the game list), making it harder to see, harder to appreciate the icon; it is less aesthetic now, since it is so small compared to the title text. Game icons were the perfect size before, at around 64x64 pixels.

6. Game icons have rounded corners, altering the icon. This is egregious in my honest opinion as it ruins the look of old game icons, needlessly "modernizing" them in a way that detracts from the original aesthetic. Why would I want my retro game icons to be modernized or altered in any way? (Not that I can really see it anyway since the icon is so small!)

7. Details panel in "Details View" was moved from the right to the left; I preferred it on the right, with the game's description on the left.

I'm afraid I will never wish to update Harmony from version 2.57 unless these changes are reverted back to how they used to be, or at least become optional via ThemeModifier.

You might recommend to just move to another theme, but I've tried just about every theme available in Playnite's "Add-ons > Browse > Themes Desktop" section, and Harmony 2.57 is the only one that bothers me the least (it's the only one I like with Platform Banners in Grid View, and Platform Banners are crucial since I have a lot of emulated games). Yet, the updates ruined even this great theme.

Honestly, I *would* be fine with never updating, ideally, but it's a real worry that the theme might become unsupported in a future version of Playnite if I never update it, which is what prompted me to make this issue "feature request" in the first place. Also, it will be a bit of a hassle to go through the addon update list and uncheck Harmony every single time I wish to update addons.

Even if you can't add ThemeModifier options, I saw that you have a theme called Stardust with a "Legacy Layout" version. Maybe you could at least provide a "Harmony (Legacy Layout)" theme as well? That might be the simplest compromise. It would just be Harmony version 2.57 (or whatever the last one was before the drastic UI updates), but that is maintained in order to remain compatible with future Playnite updates.

Thank you for taking the time to read and thank you for making this great theme. I actually do think, after typing all of this, that making a Legacy Layout version of Harmony would be the best solution, but ThemeModifier options could perhaps work, as well, albeit it would be more work to implement on both your end and the user's end.

**Comment by AaronB2019** [comment]
That's true, I would prefer having the details panel in Details view to go back to the right side, it was the main reason why I used this theme.

**Comment by maddoodles** [comment]
A little off-topic, but are there _any_ up-to-date screenshots and setup guides? It's super weird because I have no idea what this theme is supposed to look like now, and I picked it because I liked the screenshots lol, and I'm not even sure there's an old setup for that, never mind the current version. Kinda wasted my time getting ready to set this up not gonna lie lol

**Comment by darklinkpower** [comment]
- Screenshots have been updated to reflect the current version of the theme, and I have added the used settings as well: https://github.com/darklinkpower/Harmony?tab=readme-ov-file#screenshots

- An option has been added to move the Details column to the right, can be configured using the ThemeModifier extension.

Also if anyone prefers a previous version of the theme, all releases are available here: https://github.com/darklinkpower/Harmony/releases

If someone prefers this, to prevent Playnite from updating an older version of the theme if you install it you can modify the `Id` value in the `theme.yaml` in your local theme installation using any text editor for anything else so Playnite thinks it's a different theme and doesn't try to update it.

![Image](https://github.com/user-attachments/assets/40d22757-b90e-4510-8122-2d3262b5bb96)

---

Regarding other changes, I'm open to contributions if you think they could improve the theme, preferably implemented as an option for the ThemeModifier extension if possible, since at the moment all these suggestions fall under what is described here: https://github.com/darklinkpower/Harmony?tab=readme-ov-file#a-note-on-feature-requests
